tests/XPath/XPathTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimpleSAML\Test\XPath;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use SimpleSAML\XPath\XPath;
```

Comment:
Would you mind adding some use-statements for DOMDocument, DOMElement, Throwable and the libxml-functions?